src/index.js

```diff
@@ -138,14 +211,12 @@ export function createRemarkPlugin(options = {}) {
         }
       });
 
-      dom.window.document
-        .querySelector('code')
-        .setAttribute('data-language', lang);
+      const code = dom.window.document.querySelector('code');
 
-      node.value = sanitizeHtml(
-        dom.window.document.body.innerHTML,
-        sanitizeOptions
-      );
+      code.setAttribute('data-language', lang);
+      code.setAttribute('data-theme', highlighter.themeKey);
+      highlighter.themeName && code.classList.add(highlighter.themeName);
```

Comment:
What use cases would the `class` name be for, given that there's now a `data-theme` selector?

Reply:
I think defaulting to `data-theme="default"` works in the case of a single theme JSON object/string. Then you wouldn't need to add any class names?

Reply:
Yeah, there's no real need for a class name given the change to using a key.